docs/source/mb_specification.rst

```diff
@@ -0,0 +1,143 @@
+
+A JSON schema for this file can be found at https://github.com/Project-MONAI/MONAI/blob/3049e280f2424962bb2a69261389fcc0b98e0036/monai/apps/mmars/schema/metadata.json
+
+An example JSON metadata file:
+
+::
+
+  {
+      "version": "0.1.0",
+      "changelog": {
+          "0.1.0": "complete the model package",
+          "0.0.1": "initialize the model package structure"
+      },
+      "monai_version": "0.8.0",
+      "pytorch_version": "1.10.0",
+      "numpy_version": "1.21.2",
+      "optional_packages_version": {"nibabel": "3.2.1"},
+      "task": "Decathlon spleen segmentation",
+      "description": "A pre-trained model for volumetric (3D) segmentation of the spleen from CT image",
+      "authorship": "MONAI team",
+      "copyright": "Copyright (c) MONAI Consortium",
+      "data_source": "Task09_Spleen.tar from http://medicaldecathlon.com/",
+      "data_type": "dicom",
+      "dataset_dir": "/workspace/data/Task09_Spleen",
+      "image_classes": "single channel data, intensity scaled to [0, 1]",
+      "label_classes": "single channel data, 1 is spleen, 0 is everything else",
+      "pred_classes": "2 channels OneHot data, channel 1 is spleen, channel 0 is background",
+      "eval_metrics": {
+          "mean_dice": 0.96
+      },
+      "intended_use": "This is an example, not to be used for diagnostic purposes",
+      "references": [
+          "Xia, Yingda, et al. '3D Semi-Supervised Learning with Uncertainty-Aware Multi-View Co-Training.' arXiv preprint arXiv:1811.12506 (2018). https://arxiv.org/abs/1811.12506.",
+          "Kerfoot E., Clough J., Oksuz I., Lee J., King A.P., Schnabel J.A. (2019) Left-Ventricle Quantification Using Residual U-Net. In: Pop M. et al. (eds) Statistical Atlases and Computational Models of the Heart. Atrial Segmentation and LV Quantification Challenges. STACOM 2018. Lecture Notes in Computer Science, vol 11395. Springer, Cham. https://doi.org/10.1007/978-3-030-12029-0_40"
+      ],
+      "network_data_format":{
+          "inputs": {
+              "image": {
+                  "type": "image",
+                  "format": "magnitude",
+                  "num_channels": 1,
+                  "spatial_shape": [160, 160, 160],
+                  "dtype": "float32",
+                  "value_range": [0, 1],
+                  "is_patch_data": false,
+                  "channel_def": {0: "image"}
+              }
+          },
+          "outputs":{
+              "pred": {
+                  "type": "image",
+                  "format": "segmentation",
+                  "num_channels": 2,
+                  "spatial_shape": [160, 160, 160],
+                  "dtype": "float32",
+                  "value_range": [0, 1],
```

Comment:
Hi @ericspod ,

Some internal researchers raised a question about this "outputs" sections, is it exactly the output of network or output after postprocessing? If just output of network, the value range is not [0, 1].

Thanks.

Reply:
I see, so do you want any change in this metadata config?

Thanks.

Reply:
We should discuss what this should be first if we need to go deeper into specifying things like I say, then we'll have an idea of what to put here. I'm afraid we're encountering more details we need to address as we go.